### DISCUSSION TITLE
3.11 problems

### DISCUSSION THREAD

**Discussion by julien-paris** [code4recovery/12-step-meeting-list#433] [Q&A] [answered]
Hi all, for 3.11 broke our site. 
I renamed the plugin
Wanted to roll back but unable, because I couldnt find the previous version
Finally re-enabled 3.11
the site now works but the /meetings/ page now seems to not exist, and redirects to the home page. 

Will now make a backup and try to re-install the plugin but I dont have high hopes.

Any suggestions would be welcome

Much thanks, 

Julien

**Comment by brianw-area24** [comment, marked as answer]
Hi Julien,

Likely, everything is there, and it's an easy fix. Two things to try:

1. In the Wordpress Dashboard, go to Settings->Permalinks, and save this page. Don't need to make any changes, just save it.
2. If that doesn't work, edit a meeting click on "Update", again, you don't need to make any changes, just update.

Also see Discussion Items #414 and #431 

Brian W

**Comment by julien-paris** [comment]
Hi Brian, thanks for your answer. None of those work. The meetings are there: https://alcoholics-anonymous.eu/meetings/zug-english-aa-group/

But the /meetings/ page does not appear: https://alcoholics-anonymous.eu/meetings/

Any ideas. ThanksJulien 
---------------------------------------------




 

    On Saturday, July 10, 2021, 05:00:25 PM GMT+2, brianw-area24 ***@***.***> wrote:  
 
 


Hi Julien,

Likely, everything is there, and it's an easy fix. Two things to try:
   
   - In the Wordpress Dashboard, go to Settings->Permalinks, and save this page. Don't need to make any changes, just save it.
   - If that doesn't work, edit a meeting click on "Update", again, you don't need to make any changes, just update.

Also see Discussion Items #414 and #431

Brian W

**Comment by julien-paris** [comment]
Just started working again!I did the permalinks save again. Thanks, julien ---------------------------------------------




 

    On Saturday, July 10, 2021, 05:00:25 PM GMT+2, brianw-area24 ***@***.***> wrote:  
 
 


Hi Julien,

Likely, everything is there, and it's an easy fix. Two things to try:
   
   - In the Wordpress Dashboard, go to Settings->Permalinks, and save this page. Don't need to make any changes, just save it.
   - If that doesn't work, edit a meeting click on "Update", again, you don't need to make any changes, just update.

Also see Discussion Items #414 and #431

Brian W

**Comment by julien-paris** [comment]
Hi Brian, 
Am sending a video because I am digging into a long lasting issue we have with syncing meetings, and am hoping you can help. 
Am reclo for CER (European English speaking meetings) and ex webmaster for paris, where I am still able to login and fiddle around with the settings. 

** The specific issue I have is that when i sync meeting from Paris, and Berlin, these cities are imported as regions, as opposed to being set under their respective countries - France and Germany.**

What i do to bypass the issue is: 
1. switch them to countries 
2. sync
3. move them back under their correct country. 

https://www.loom.com/share/b4e90b0df7b943f9982c6158c3fa115e [https://www.loom.com/share/b4e90b0df7b943f9982c6158c3fa115e]


This is not really best practice and the problem now happens with a new region, which has multiple cities. 

Any advice? 

Thanks, 

Julien 

On 10-Jul-21 17:00:25, brianw-area24 ***@***.***> wrote:
Hi Julien,
Likely, everything is there, and it's an easy fix. Two things to try:
* In the Wordpress Dashboard, go to Settings->Permalinks, and save this page. Don't need to make any changes, just save it.
* If that doesn't work, edit a meeting click on "Update", again, you don't need to make any changes, just update.
Also see Discussion Items #414 [https://github.com/code4recovery/12-step-meeting-list/discussions/414] and #431 [https://github.com/code4recovery/12-step-meeting-list/discussions/431]
Brian W
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub [https://github.com/code4recovery/12-step-meeting-list/discussions/433#discussioncomment-988490], or unsubscribe [https://github.com/notifications/unsubscribe-auth/ARSTAQUAHYSFC32OMKDMJKDTXBOARANCNFSM5AEJYA6A].

-- 
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus

**Reply by tech2serve** [reply]
Hi Julian, Please open a new Q&A discussion for this question. I looked for a way to do that, but GitHub doesn’t appear to provide that feature.

**Reply by julien-paris** [reply]
Thx, all done
